### PR TITLE
7476: show complete QC button on document viewer screen

### PR DESCRIPTION
**Pull request by rachaelparris**
Screen Shot 2021-02-18 at 12 51 06 PM

### CODE REVIEW

File: shared/src/business/utilities/getFormattedCaseDetail.js [hunk 1]
```diff
@@ -151,6 +151,9 @@ const formatDocketEntry = (applicationContext, docketEntry) => {
   formattedEntry.qcWorkItemsUntouched =
     qcWorkItem && !qcWorkItem.isRead && !qcWorkItem.completedAt;
 
+  formattedEntry.qcNeeded =
+    formattedEntry.qcWorkItemsUntouched && !formattedEntry.isInProgress;
```
Comment (matthopson) [inline shared/src/business/utilities/getFormattedCaseDetail.js:155]: We have some logic that is similar in `formattedCaseDetail` - is that at all related?

Comment (rachaelparris) [reply shared/src/business/utilities/getFormattedCaseDetail.js:155]: Yeah, we removed all of that and combined these all together to be in one place. Unless there are some we missed, which I would be happy to remove 😄

Comment (matthopson) [reply shared/src/business/utilities/getFormattedCaseDetail.js:155]: Oh snap, I should have kept scrolling... I saw some github suggestions and thought I was at the end...  Thank you for doing this. These two files confuse me CONSTANTLY.

**Review by matthopson — APPROVED** [review]
